pennylane/transforms/convert_to_numpy_parameters.py

```diff
@@ -81,9 +85,16 @@ def convert_to_numpy_parameters(circuit: QuantumScript) -> QuantumScript:
     False
 
     """
-    new_ops = (_convert_op_to_numpy_data(op) for op in circuit.operations)
-    new_measurements = (_convert_measurement_to_numpy_data(m) for m in circuit.measurements)
-    new_circuit = circuit.__class__(
-        new_ops, new_measurements, shots=circuit.shots, trainable_params=circuit.trainable_params
+    new_ops = (_convert_op_to_numpy_data(op) for op in tape.operations)
+    new_measurements = (_convert_measurement_to_numpy_data(m) for m in tape.measurements)
+    new_circuit = tape.__class__(
+        new_ops, new_measurements, shots=tape.shots, trainable_params=tape.trainable_params
     )
-    return new_circuit
+
+    def null_postprocessing(results):
```

Comment:
Optional extension to this would be using the postprocessing function to return results to the correct interface.

See the `ResultBatch` -> `ResultBatch` functions:

https://github.com/PennyLaneAI/pennylane/blob/332d09007f885f1e8ae06fbe99b7112d45f840d9/pennylane/workflow/interfaces/jax.py#L213

https://github.com/PennyLaneAI/pennylane/blob/332d09007f885f1e8ae06fbe99b7112d45f840d9/pennylane/workflow/interfaces/jax_jit.py#L53

https://github.com/PennyLaneAI/pennylane/blob/332d09007f885f1e8ae06fbe99b7112d45f840d9/pennylane/workflow/interfaces/torch.py#L239

https://github.com/PennyLaneAI/pennylane/blob/332d09007f885f1e8ae06fbe99b7112d45f840d9/pennylane/workflow/interfaces/tensorflow.py#L134

But those probably have enough patches and fixes we should probably keep that step localized to those functions.